src/guides/v2.3/release-notes/release-notes-2-3-4-open-source.md

```diff
@@ -1508,11 +1510,11 @@ We have fixed hundreds of issues in the Magento 2.3.4 core code.
 
 <!--- MC-17014-->
 
-*  User can now successfully save a fixed product tax  (FPT) to a product that is assigned to a specific website.
+*  You can now successfully save a fixed product tax  (FPT) to a product that is assigned to a specific website.
```

Comment:
Same dupe spaces here and next line.

Reply:
Thank you, but no time to groom for double spaces that do not affect meaning or display.